lib/services/apiAppManagement/README.md

```diff
@@ -0,0 +1,33 @@
+# Microsoft Azure SDK for Node.js - ApiApp Management
+
+This project provides a Node.js package that makes it easy to manage Microsoft ApiApp. Right now it supports:
+- **Node.js version: 0.6.15 or higher**
+- **API version: 2015-01-14**
+
+## Features
+
+ - TODO
```

Comment:
I assume, because it is the initial cut, so the TODO here are expected and will be addressed later on. Otherwise, please fill in the content.